modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -26,6 +26,34 @@ class NSFVBase : public BaseType
 
   NSFVBase(const InputParameters & parameters);
 
+  ///@{ general public interface functions
+  bool hasEnergyEquation() const { return _has_energy_equation; }
+  bool hasScalar() const { return _has_scalar_equation; }
+  unsigned int dim() const { return _dim; }
+  std::string rhieChowName() const
+  {
+    if (_porous_medium_treatment)
+      return "pins_rhie_chow_interpolator";
+    return "ins_rhie_chow_interpolator";
+  }
+  ///@}
+
+  ///@{ public interface for variable and property names
+  std::string pressureName() const { return _pressure_name; }
```

Comment:
use the specific type (NonLinearVariableName iirc)

Reply:
```suggestion
  NonlinearVariableName pressureName() const { return _pressure_name; }
```